README2.md

```diff
@@ -0,0 +1,19 @@
+# Create a Classic Cluster
```

Comment:
Bingo. Having another readme to reference in the previous one is a good idea. +1.